### DISCUSSION TITLE
Using unleash token as env var

### DISCUSSION THREAD

**Discussion by ep4sh** [Unleash/unleash#7351] [Q&A]
Dear community,
could you please advise, is there a special/magic env variable I can assign to the Unleash token for my app?
My application lives in Kubernetes, and I'm supposed using Kubernetes secrets (as env vars is a preferable way)

**Comment by chriswk** [comment]
Hi @ep4sh , you don't say which SDK/app you're using or how you're deploying your app to Kubernetes.

We're happy to help, but we need more details on what you're actually trying to do
